third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -333,11 +333,6 @@ impl InterpreterImpl<'_> {
                         current_frame = frame;
                         current_frame.pc += 1; // advance past the Call instruction in the caller
                     } else {
-                        // end of execution. `self` should no longer be used afterward
```

Comment:
Could you explain the reason the code being removed here?  Why did we have this code given this was already called above?